Upstream: reported
Url: https://github.com/KDAB/GammaRay/issues/608
Reason: The upgrade to Qt 5.15 requires some changes
diff --git a/plugins/network/networksupport.cpp b/plugins/network/networksupport.cpp
index 32191d0..516eefe 100644
--- a/plugins/network/networksupport.cpp
+++ b/plugins/network/networksupport.cpp
@@ -89,7 +89,6 @@ Q_DECLARE_METATYPE(QNetworkInterface::InterfaceFlags)
 #endif
 Q_DECLARE_METATYPE(QNetworkProxy::Capabilities)
 Q_DECLARE_METATYPE(QNetworkProxy::ProxyType)
-Q_DECLARE_METATYPE(QSocketNotifier::Type)
 #ifndef QT_NO_SSL
 Q_DECLARE_METATYPE(QSsl::KeyAlgorithm)
 Q_DECLARE_METATYPE(QSsl::KeyType)
diff --git a/plugins/widgetinspector/overlaywidget.h b/plugins/widgetinspector/overlaywidget.h
index 5e0bb13..5acd12c 100644
--- a/plugins/widgetinspector/overlaywidget.h
+++ b/plugins/widgetinspector/overlaywidget.h
@@ -32,6 +32,7 @@
 #include <QLayout>
 #include <QPointer>
 #include <QWidget>
+#include <QPainterPath>
 
 namespace GammaRay {
 
